src/components/page-list/page-list.jsx

```diff
@@ -75,7 +75,7 @@ export default class PageList extends React.Component {
     const onClick = this.didClickRow.bind(this, record);
 
     return (
-      <tr key={record.uuid} onClick={onClick}>
+      <tr styleName="page-list" key={record.uuid} onClick={onClick}>
```

Comment:
Is there a reason you added this? With the current CSS, it gets ignored (`table-layout: fixed` is only valid on things with `display: table`) and if we added more to the `.page-list` CSS class I don’t think we’d actually want it to apply to both this and the table as a whole.

Reply:
The table tag still has styleName='page-list' but seem to not apply to this particular column (URL). Other reason could be (might be true or not ) is that a URL does not have any spaces or separators (apart from the '-' ) in it. So maybe the URL cannot be broken down or wrapped around to the next line. Like in the image above the URL is not broken or does not go into the next line until the '-' symbol is encountered.

Reply:
Let me know if https://github.com/edgi-govdata-archiving/web-monitoring-ui/pull/414/commits/5ee9cb72fa1856607a6f5f423a259ed400120a0d fixes this.

Reply:
Looks good!